articles/automatic-software-install-in-fleet.md

```diff
@@ -0,0 +1,82 @@
+# Automatic policy-based installation of software on hosts
+
+![Top Image](../website/assets/images/articles/automatic-software-install-top-image.png)
+
+Fleet [v4.57.0](https://github.com/fleetdm/fleet/releases/tag/fleet-v4.57.0) introduces the ability to automatically and remotely install software on hosts based on predefined policy failures. This guide will walk you through the process of configuring fleet for automatic installation of software on hosts using previously uploaded installation images and based on previously programmed policies.  You'll learn how to configure and use this feature, as well as understand how the underlying mechanism works.
```

Comment:
What do you mean by "previously uploaded installation images and based on previously programmed policies"?